frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,53 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<string>
+     */
+    public static function getFrequentsByLevel(
+        string $problemLevel
+    ) {
+        $sql = '
+            SELECT
+                t.name
+            FROM
+                Problems_Tags pt
+            INNER JOIN
+                Tags t ON t.tag_id = pt.tag_id
+            WHERE
+                pt.problem_id
+            IN (
+                SELECT
+                    problem_id
+                FROM
+                    Problems_Tags
+                WHERE
+                    tag_id = (
+                        SELECT
+                            tag_id
+                        FROM
+                            Tags
+                        WHERE name = ? )
+            ) AND
+                name LIKE "problemTag%"
+            GROUP BY
+                t.name
+            ORDER BY
+                COUNT(pt.problem_id)
+            ';
+
+        $results = [];
+        /** @var array{name: string} row */
+        foreach (
+            \OmegaUp\MySQLConnection::getInstance()->GetAll(
+                $sql,
+                [ $problemLevel ]
+            ) as $row
+        ) {
+            $results[] = $row['name'];
+        }
+
+        return $results;
```

Comment:
Aquí puedes regresar lo que se obtiene de la consulta, ya que es un arreglo asociativo tal cual se requiere en el API, de esta forma ya no tienes que hacer conversiones ni aquí, ni en el API. Algo así:
```suggestion
        /** @var list<array{name: string}> */
        return \OmegaUp\MySQLConnection::getInstance()->GetAll(
            $sql,
            [$problemLevel]
        );
```